### PR TITLE
 - THe short name will now be logged and an exception will not be thr…

### PR DESCRIPTION
**Pull request by Michaellh0079**
…own if no granules exist at the provided path.

### CODE REVIEW

File: task/main.py [hunk 1]
```diff
@@ -41,33 +44,34 @@ def __init__(self, event):
...
-        self.logger = CumulusLogger(name='Recursive-Discover-Granules', level=logging_level)
 
     def discover(self):
         """
         Helper function to kick off the entire discover process
         """
         granules = self.collection.get('meta', {}).get('granules', None)
         if granules:
-            self.logger.info(f'Received {len(granules)} to reingest.')
+            logger.info(f'Received {len(granules)} to reingest.')
```
Comment (amarouane-ABDELHAK) [inline task/main.py:54]: Can you add the collection name to the log?

File: task/main.py [hunk 2]
```diff
@@ -41,33 +44,34 @@ def __init__(self, event):
...
-            self.logger.info(f'Discovered {len(granule_dict)} granules.')
+            if not granule_dict:
+                logger.warning(f'Warning: Found 0 granules at the provided location.')
+            else:
+                logger.info(f'Discovered {len(granule_dict)} granules for update processing.')
             self.check_granule_updates_db(granule_dict)
 
-            self.logger.info(f'{len(granule_dict)} granules for ingest.')
             output = self.cumulus_output_generator(granule_dict)
+            print(f'Returning cumulus output for {len(output)} {self.collection.get("name")} granules.')
```
Comment (Michaellh0079) [inline task/main.py:69]: @amarouane-ABDELHAK The collection name is printed after the cumulus output is generated. I can add above as well if needed?

File: task/main.py (hunk 1 above)
Comment (Michaellh0079) [reply task/main.py:54]: https://github.com/ghrcdaac/discover-granules-tf-module/pull/49/files/b4c32407c019b95a31e8259cd419e68c1b37ef27#r770729224

File: task/main.py (hunk 2 above)
Comment (amarouane-ABDELHAK) [reply task/main.py:69]: Yes please add it to see how many are discovered (or none discovered)

File: task/main.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 import os
 import re
 import time
+from math import floor
```
Comment (amarouane-ABDELHAK) [inline task/main.py:5]: Are we using floor?

**Review by amarouane-ABDELHAK — APPROVED** [review]

**Review by amarouane-ABDELHAK — APPROVED** [review]

**Pull request merged**